test/adapter/json_api/links_test.rb

```diff
@@ -0,0 +1,37 @@
+require 'test_helper'
+
+LinkPost = Class.new(Model)
+class LinkPostSerializer < ActiveModel::Serializer
+  belongs_to :link_author
+end
+
+LinkAuthor = Class.new(Model)
+class LinkAuthorSerializer < ActiveModel::Serializer
+  links do
+    link :self, href: "//link_author/#{object.id}"
```

Comment:
It's in the roadmap we defined for 0.10, so I'm not sure what you mean by 'not now'.

As for automatically building links, this would be nice, but wouldn't cover use-cases where you have to give a custom link (not the `self`, but the JSON API spec allows for other links).